### PR TITLE
add switch atomic component

### PR DESCRIPTION
**Pull request by pietro-maximoff**
https://sovryn.monday.com/boards/2218344956/pulses/2263336228

### CODE REVIEW

**Review by soulBit — APPROVED** [review]

**Comment by rick23p** [comment]
If ```innerLabelChecked``` is empty and ```innerLabel``` has value (or vice versa ) then on toggle the checkbox jumps up and down

File: src/app/components/Switch/index.tsx [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    <div
+      className={classNames(className, {
+        'tw-inline-block tw-align-middle': inline,
+      })}
+    >
+      <label
+        className={classNames(
+          styles.switch,
+
+          { 'tw-cursor-not-allowed tw-opacity-30': disabled },
```
Comment (rick23p) [inline src/app/components/Switch/index.tsx:47]: let's use "tw-opacity-50" to be more consistent with other atomic components

**Comment by rick23p** [comment]
Tested on Safari: (not aligned)
Screen Shot 2022-03-15 at 3 57 15 PM

**Comment by rick23p** [comment]
I think giving it a "tw-select-none" might be good to avoid the text being selected on clicks
Screen Shot 2022-03-15 at 3 59 24 PM

**Review by rick23p — CHANGES_REQUESTED** [review]

**Comment by JoeStorm99** [comment]
The Switch is not properly aligned on Safari 

Screenshot 2022-03-15 at 13 15 21

**Review by JoeStorm99 — CHANGES_REQUESTED** [review]
please read the comments below, the switch is not properly aligned on Safari. 
I have also attached a screenshot.

**Review by rick23p — APPROVED** [review]

**Review by soulBit — DISMISSED** [review]

**Comment by JoeStorm99** [comment]
Well-aligned on safari and all working well 
Screenshot 2022-03-16 at 10 51 46

**Review by JoeStorm99 — APPROVED** [review]
now working as expected on all browsers including Safari

**Review by Alitasovryn — APPROVED** [review]

**Comment by ShynRou** [comment]
This component will need to be updated again, the component currently completely ignores the existing switch design. The inner labels also don't work particularly well. I'm sorry the spec may not have made it clear enough that it should match the currently implemented design. (CSS override for the blueprint version).

**Comment by ShynRou** [comment]
I will do the once over as a new PR.

**Pull request merged**